README.md

```diff
@@ -32,3 +32,95 @@ For VS Code, the following VSC extensions are recommended (VSC will prompt to in
 # Acknowledgement
 
 Implementation is an extension of SimpleDB from Edward Sciore. Project is used solely for educational purpose.
+
+
+# Queries to run
+
+## Create tables with 50 - 100 rows, some duplicate rows
+Done in the CreateStudentDB.java file
+
+## Single Table Queries
+
+### SELECT
+
+`select sid, sname from student`
+
+### WHERE (equality)
+
+`select sid, sname from student where sname = 9`
+
+### WHERE (non equality)
+
+`select sid, sname from student where sname > 'bob'`
+
+### Search using index
+
+#### Btree Index
+
+`select sid, majorid from student where sid = 10`
+
+#### Hash Index
+
+`select sid, majorid from student where majorid = 20`
+
+### ORDER BY
+
+#### Single ORDER BY
+
+`select cid, title, deptid from course order by cid`
+
+#### Multiple ORDER BY
+
+`select cid, title, deptid from course order by cid desc, title, deptid desc`
+
+### Group By
+
+#### Single field group by
+`select yearoffered from section group by yearoffered`
+
+#### Multiple field group by
+`select yearoffered, courseid from section group by yearoffered, courseid`
+
+### Aggregates
+
+#### Single Aggregate
+`select avg(eid) from enroll`
+
+#### Multiple Aggregates
+`select avg(eid), min(studentid), sum(sectionid) from enroll`
+
+#### Group by and aggregate
+`select studentid, count(eid) from enroll group by studentid`
+`select prof, courseid, min(yearoffered), max(yearoffered) from section group by prof, courseid`
+
+### Distinct
+
+`select distinct yearoffered from section`
+`select distinct majorid, gradyear from student`
+
+### Multiple conditions
+`select gradyear, count(sid) from student where gradyear > 2020 and sname > 'bob' group by gradyear order by countofsid`
+
+## Two table joins
```

Comment:
Another query to add will be non-equi join. (FYI: this will default to nested loop join)
`select sid,sname,grade from student,enroll where sid>eid`